packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -30,6 +31,11 @@ export type ToasterSettings = {
     | 'bottom'
     | 'bottom-start'
     | 'bottom-end';
+  /**
+   * If using React v18, use`ReactDom.createRoot`function to return root and enable v18.
+   * Otherwise, old `ReadDom.render` method will be used to render toasts.
```

Comment:
Typos
```suggestion
   * If using React v18, use `ReactDOM.createRoot` function to return root and enable v18.
   * Otherwise, old `ReactDOM.render` method will be used to render toasts.
```

Reply:
removed